app/views/admin/invoices/show.html.erb

```diff
@@ -1,7 +1,14 @@
 <h1> Invoice #<%= @invoice.id %></h1>
 
 <div id="invoice-summary">
-  <p> Status: <%= @invoice.status %> </p>
+  <div id="update-invoice">
+    <p> <%= form_with(model: [ :admin, @invoice ]) do |f| %> </p>
+      <p> Status: <%= f.select(:status, [['In Progress', 0], ['Completed', 1], ['Cancelled', 2]], 
```

Comment:
This looks great!! I'm looking forward to using this for reference in the future.